erigon-lib/kv/mdbx/kv_mdbx_temporary.go

```diff
@@ -37,7 +37,7 @@ func NewTemporaryMdbx(ctx context.Context, tempdir string) (kv.RwDB, error) {
 		return &TemporaryMdbx{}, err
 	}
 
-	db, err := New(kv.ChainDB, log.Root()).InMem(path).Open(ctx)
+	db, err := New(kv.ChainDB, log.Root()).InMem(path).GrowthStep(64 * datasize.MB).Open(ctx)
```

Comment:
Sometimes we get `MDBX_MAP_FULL` error if there are lots of blocks to hold temporarily, this is to fix it